tests/smoke/desiredstate_test.go

```diff
@@ -94,9 +97,13 @@ func TestDesiredState(t *testing.T) {
 	require.NoError(t, err)
 	tests.RequireOutputContains(t, output, "The installation is out-of-sync, running the upgrade action")
 
-	installation, err := test.ShowInstallation("operator", "mybuns")
+	displayInstallation, err := test.ShowInstallation("operator", "mybuns")
 	require.NoError(t, err)
-	require.Equal(t, float64(3), installation.Parameters["log_level"])
+	for name, value := range displayInstallation.Parameters {
```

Comment:
Since Parameters is still a map, I think you can keep what was there originally instead of iterating now?